build.psake.ps1

```diff
@@ -134,6 +134,41 @@ Task Sign -depends BuildImpl -requiredVariables SettingsPath, SignScripts {
     }
 }
 
+Task Analyze -depends Build -requiredVariables CodeAnalysisStop, OutDir {
```

Comment:
OK, that's similar to what I've done in my own psake scripts before, so that's good to know. I'll do that.